spongeauth/spongeauth/urls.py

```diff
@@ -38,6 +39,7 @@
     url(r'^sso/', include(sso.urls, 'sso')),
     url(r'^$', index, name='index'),
     url(r'^api/', include(api.urls, 'api')),
+    url(r'^analytics/', analytics_index, name='analytics')
```

Comment:
Since this isn't a base route for an app, this should be `r'^analytics/$'`

Reply:
Oh thought I did this.. anyway, fixed now :+1: